dev/io.openliberty.org.jboss.narayana.rts/bnd.bnd

```diff
@@ -22,8 +22,21 @@ Export-Package: \
 
```

Comment:
I can't see any bundles which import these packages?? Are these exports actually needed, should the packages be private instead?

Reply:
They are imported implicitly by the buildpath statement in io.openliberty.microprofile.lra.1.0.internal